src/bin/simhash-find-all.cpp

```diff
@@ -0,0 +1,204 @@
+#include <iostream>
+#include <unordered_set>
+#include <sstream>
+#include <fstream>
+
+#include <getopt.h>
+
+#include "simhash.h"
+
+void usage(int argc, char** argv)
+{
+    std::cout << "usage: " << argv[0]
+              << " --blocks BLOCKS"
+              << " --distance DISTANCE"
+              << " --input INPUT"
+              << " --output OUTPUT"
+              << std::endl
```

Comment:
Sure, but this is a usage string and so it's not a hot loop. Isn't `std::endl` more idiomatic unless it's performance-critical?